Week1/Section2/Exercise3/Source3.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+Exercise 3 - Positive, Negative, or Zero
+Write a program that will ask the user for a number and determine if it is a positive number, a negative number, or zero.
+
+Example output
+Please enter a number: 5
+
+Your number is positive!
+*/
+#include <iostream>
+
+int main() {
+	int value;
+
+	std::cout << "Please enter a number: ";
+	std::cin >> value;
+
+	if (value == 0) {
+		std::cout << "Your number is Zero" << std::endl;
+	}
+	else if (value > 0) {
+		std::cout << "Your number is Positive" << std::endl;
+	}
+	else if (value < 0) {
```

Comment:
This works well, but keep in mind that this is the only possible scenario left, so an `else` would be more than enough without implying that there are more possible scenarios :)